packages/loot-core/src/server/schedules/find-schedules.ts

```diff
@@ -392,9 +394,11 @@ export async function findSchedules() {
     },
   );
 
-  let finalized = [];
+  const finalized = [];
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** 
```suggestion
  const finalized: Awaited<ReturnType<typeof findStartDate>> = [];
```

and then you should be able to remove the manual type coercion at the return statement.